doc/cue_start.dox

```diff
@@ -0,0 +1,283 @@
+﻿/**
+* @page cue_start Cue Node Getting Started Guide
+*
+* @section tableofcontents Table of Contents
```

Comment:
we don't need to add a * into the beginning of every line. It would be much easier to edit this text if those stars are not there.
If you know how to script this to remove all the stars, please do that, otherwise you can leave it to me.

Reply:
I can pull the * characters; I based this on the openmrn getting started so that's probably some sort of artifact from that work somehow.